tmt-app/src/Components/MajorPage.jsx

```diff
@@ -1,10 +1,38 @@
-import React from "react";
+import { Container, Typography } from "@mui/material";
+import React, { useEffect, useState } from "react";
 import { useParams } from "react-router-dom";
+import { MAJORS_LIST } from "../constants/careerMajorList";
 
 function MajorPage() {
-  const majorChoice = useParams();
+  const params = useParams();
+  const [data, setData] = useState([]);
 
-  return <div>{majorChoice.major}</div>;
+  const searchData = (target) => {
+    for (let i = 0; i < MAJORS_LIST.length; i++) {
+      if (target === MAJORS_LIST[i].key) {
+        setData(MAJORS_LIST[i]);
+        console.log(MAJORS_LIST[i]);
+      }
```

Comment:
Is there another method other than a for loop that will get you the data you need much more easily?

Reply:
I will try to look for another method